docs/DeveloperGuide.md

```diff
@@ -35,4 +348,212 @@
 
 ## Instructions for manual testing
 
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+The following are instructions for testers to manual test:
+
+### Launch and shutdown
+#### Initial Launch
+- Ensure that Java 11 is installed on your device.
+- Download the JAR file and copy into an empty folder
+- Open the command terminal on your device.
+- Navigate to the folder in command terminal and run the command `java -jar [filename].jar`
+- Alternatively, double click on the JAR file to run the app.
+#### Adding a record
+1. Adding an expenditure
```

Comment:
Not sure why but the bullet point for this is not showing up correctly on the github pages, but you could possibly include a new line before this, which could fix the error.
![image](https://user-images.githubusercontent.com/88092096/229010975-75b0fba2-917c-43f8-ba49-d5d5757b2a68.png)